apps/projects/contracts/Projects.sol

```diff
@@ -230,44 +279,43 @@ contract Projects is IsContract, AragonApp {
 
     /**
      * @notice Get issue data from the registry.
-     * @param _repoId The id of the Github repo in the projects registry
+     * @param _repoId The id of the repo in the projects registry
      */
-    function getIssue(bytes32 _repoId, uint256 _issueNumber) external view
-    returns(bool hasBounty, uint standardBountyId, bool fulfilled, uint balance, address token, string dataHash, address assignee)
+    function getIssue(bytes32 _repoId, uint256 _issueNumber) external view isInitialized
+    returns(bool hasBounty, uint standardBountyId, bool fulfilled, uint balance, address assignee)
```

Comment:
@Quazia I see that `getIssue` is no longer returning `token` or `dataHash`. In the frontend, we relied on this `dataHash` value to look up extra info from IPFS. What is the new way for the frontend to find this hash?